### PR TITLE
Pic 2880

### PR DESCRIPTION
**Pull request by johnyfarrar**

### CODE REVIEW

File: server/routes/handlers/getCaseOutcomesRouteHandler.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const getCaseOutcomesRouteHandler = () => async (req, res) => {
```
Comment (rschallapalli) [inline server/routes/handlers/getCaseOutcomesRouteHandler.js:1]: unit test?

File: integration-tests/integration/features/outcomes.feature [hunk 2]
```diff
@@ -12,4 +12,12 @@ Feature: Outcomes List
...
     When I click on the "Outcomes" link in the Primary navigation
-    And the page should produce a 404 error
\ No newline at end of file
+    Then I should be on the "Hearing outcomes" page
+
+    And I should see the phase banner
+    And I should see the tag "Feedback"
+    And I should see phase banner link "Give us your feedback" with href "https://docs.google.com/forms/d/e/1FAIpQLScluoDOXsJ_XBO3iOp283JE9mN3vTVNgEJcPNDHQQvU-dbHuA/viewform?usp=sf_link"
+    
+    And I should see a tab with text "Cases to result"
```
Comment (rschallapalli) [inline integration-tests/integration/features/outcomes.feature:21]: Also add - on clicking the tab should land on corresponding location?

**Comment by rschallapalli** [comment]
couple of minor comments

File: integration-tests/integration/features/outcomes.feature (hunk 2 above)
Comment (johnyfarrar) [reply integration-tests/integration/features/outcomes.feature:21]: Have added that now

File: server/routes/handlers/getCaseOutcomesRouteHandler.js (hunk 1 above)
Comment (johnyfarrar) [reply server/routes/handlers/getCaseOutcomesRouteHandler.js:1]: Added test!

**Review by rschallapalli — APPROVED** [review]

**Pull request merged**